charging_area:
  upper_left: [-1.5,1.5]
  down_right: [1.5,-1.5]

landing_areas:
  # - name: box1
  #   coords:
  #     - [1.00138695, -0.2763554, 2.24064088]
  #     - [1.00138695,  0.0236446, 2.24064088]
  #     - [0.70138695, -0.2763554, 2.24064088]
  #     - [0.70138695,  0.0236446, 2.24064088]

  - name: box2
    coords:
      - [-6.21807749, -1.23965955, 2.05602407]
      - [-6.21807749, -1.53965955, 2.05602407]
      - [-5.91807749, -1.23965955, 2.05602407]
      - [-5.91807749, -1.53965955, 2.05602407]

  - name: box3
    coords:
      - [7.89766867, 2.65825594, 2.37501574]
      - [7.89766867, 2.95825594, 2.37501574]
      - [7.59766867, 2.65825594, 2.37501574]
      - [7.59766867, 2.95825594, 2.37501574]

  - name: box4
    coords:
      - [-3.57053249, 6.79232586, 1.79399443]
      - [-3.57053249, 7.09232586, 1.79399443]
      - [-3.87053249, 6.79232586, 1.79399443]
      - [-3.87053249, 7.09232586, 1.79399443]

  - name: box5
    coords:
      - [4.59965102, -4.20259369, 1.7023772]
      - [4.59965102, -4.50259369, 1.7023772]
      - [4.89965102, -4.20259369, 1.7023772]
      - [4.89965102, -4.50259369, 1.7023772]

  - name: box6
    coords:
      - [7.86800449, 8.86130011, 1.92914629]
      - [7.86800449, 8.56130011, 1.92914629]
      - [8.16800449, 8.86130011, 1.92914629]
      - [8.16800449, 8.56130011, 1.92914629]

  - name: box7
    coords:
      - [4.53593366, 1.67754698, 1.06291938]
      - [4.53593366, 1.37754698, 1.06291938]
      - [4.83593366, 1.67754698, 1.06291938]
      - [4.83593366, 1.37754698, 1.06291938]

  - name: box8
    coords:
      - [-2.96364172, 7.35844671, 2.94387102]
      - [-2.96364172, 7.05844671, 2.94387102]
      - [-2.66364172, 7.35844671, 2.94387102]
      - [-2.66364172, 7.05844671, 2.94387102]

  - name: box9
    coords:
      - [-7.50100071, -8.25461166, 2.30138183]
      - [-7.80100071, -8.25461166, 2.30138183]
      - [-7.50100071, -8.55461166, 2.30138183]
      - [-7.80100071, -8.55461166, 2.30138183]

  - name: box10
    coords:
      - [1.1000681, -6.88320668, 1.74859047]
      - [1.4000681, -6.88320668, 1.74859047]
      - [1.1000681, -6.58320668, 1.74859047]
      - [1.4000681, -6.58320668, 1.74859047]
